plugin/web/extensions/storytelling/modals/sceneeditor/components/index.tsx

```diff
@@ -0,0 +1,82 @@
+import { styled } from "@web/theme";
```

Comment:
Please rename the directory `sceneEditor` (storytelling/modals/sceneEditor/....)

Reply:
Updated.